external/mmsegmentation/segmentation_tasks/apis/segmentation/ote_utils.py

```diff
@@ -59,13 +59,8 @@ def on_epoch_end(self, epoch, logs=None):
         self._calculate_average_epoch()
         score = None
         if hasattr(self.update_progress_callback, 'metric') and isinstance(logs, dict):
-            score = logs.get(self.update_progress_callback.metric, None)
-
-        # Workaround for NNCF trainer, which uses callback of a different type.
-        if score is not None:
-            self.update_progress_callback(self.get_progress(), score=float(score))
-        else:
-            self.update_progress_callback(int(self.get_progress()))
+            score = float(logs.get(self.update_progress_callback.metric, None))
```

Comment:
Is that possible that None is casted to float here?

Reply:
Yes, will fix